appinventor/components-ios/src/Chart.swift

```diff
@@ -249,6 +249,34 @@ import DGCharts
       }
     }
   }
+  
+  @objc open func ExtendDomainToInclude(_ x: Double) {
+    if let chartView = (_chartView as AnyObject) as? AxisChartView {
```

Comment:
I'm not sure why the cast to AnyObject is necessary here. It should be sufficient to just perform the optional downcast since _chartView is already a type in the class hierarchy including AxisChartView.

I'd also recommend switching to a guard let.